pkg/certs/certs_test.go

```diff
@@ -94,54 +94,54 @@ var _ = Describe("Keypair generation", func() {
 		notAfter := time.Now().Add(-10 * time.Hour)
 		notBefore := notAfter.Add(-90 * 24 * time.Hour)
 		ca, err := createCAWithValidity(notBefore, notAfter, nil, nil, "root", "namespace")
-		Expect(err).To(BeNil())
+		Expect(err).ToNot(HaveOccurred())
 		isExpiring, _, err := ca.IsExpiring()
 		Expect(isExpiring, err).To(BeTrue())
 	})
 
 	It("doesn't marks a valid certificate as expiring", func() {
 		ca, err := CreateRootCA("test", "namespace")
-		Expect(err).To(BeNil())
+		Expect(err).ToNot(HaveOccurred())
 		isExpiring, _, err := ca.IsExpiring()
 		Expect(isExpiring, err).To(BeFalse())
 	})
 
 	When("we have a CA generated", func() {
 		It("should successfully generate a leaf certificate", func() {
 			rootCA, err := CreateRootCA("test", "namespace")
-			Expect(err).To(BeNil())
+			Expect(err).ToNot(HaveOccurred())
 
 			pair, err := rootCA.CreateAndSignPair("this.host.name.com", CertTypeServer, nil)
-			Expect(err).To(BeNil())
+			Expect(err).ToNot(HaveOccurred())
 
 			cert, err := pair.ParseCertificate()
-			Expect(err).To(BeNil())
+			Expect(err).ToNot(HaveOccurred())
 
 			key, err := pair.ParseECPrivateKey()
-			Expect(err).To(BeNil())
+			Expect(err).ToNot(HaveOccurred())
 
 			Expect(cert.PublicKey).To(BeEquivalentTo(&key.PublicKey))
 			Expect(cert.IsCA).To(BeFalse())
 			Expect(cert.BasicConstraintsValid).To(BeTrue())
-			Expect(cert.KeyUsage & x509.KeyUsageDigitalSignature).To(Not(Equal(0)))
-			Expect(cert.KeyUsage & x509.KeyUsageKeyEncipherment).To(Not(Equal(0)))
+			Expect(cert.KeyUsage & x509.KeyUsageDigitalSignature).ToNot(BeZero())
+			Expect(cert.KeyUsage & x509.KeyUsageKeyEncipherment).ToNot(BeZero())
 			Expect(cert.ExtKeyUsage).To(Equal([]x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth}))
 			Expect(cert.NotBefore).To(BeTemporally("<", time.Now()))
 			Expect(cert.NotAfter).To(BeTemporally(">", time.Now()))
-			Expect(cert.VerifyHostname("this.host.name.com")).To(BeNil())
+			Expect(cert.VerifyHostname("this.host.name.com")).ToNot(HaveOccurred())
```

Comment:
bad semantics here. Probably better .To(Succeed()) or such. Marking to look later.